File: backend/database/seeders/UserSeeder.php
<?php

namespace Database\Seeders;

use Illuminate\Database\Seeder;
use App\Services\UserService;

class UserSeeder extends Seeder
{
    public function run()
    {
        for ($i = 0; $i < 30; $i++) {
            UserService::registerUser("User {$i}", "user{$i}@example.com", 'password');
        }
    }
}

File: kkn_backend/database/seeders/user_seeder.go
package seeders

import (
	"fmt"
	"kkn_backend/app/services"
)

func SeedUsers() error {
	amount_to_generate := 30
	for i := 0; i < amount_to_generate; i++ {
		_, err := services.RegisterUser(fmt.Sprintf("User %d", i), fmt.Sprintf("user%d@example.com", i), "password")
		if err != nil {
			return err
		}
	}
	return nil
}
